front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -147,7 +161,34 @@ const FullscreenModal = ({
         contentBgColor={contentBgColor}
       >
         <StyledFocusOn autoFocus>
-          {topBar}
+          {modalTitle && (
+            <Box
+              bgColor={colors.white}
+              borderBottom={`1px solid ${colors.grey300}`}
+              display="flex"
+              alignItems="center"
+              justifyContent="center"
+              position="relative"
+            >
+              <Title
+                as={titleAs || 'h2'}
+                variant={titleVariant || 'h5'}
+                m="0"
```

Comment:
Default margins are unequal on y-axis, so resetting them and using padding instead. Could have also used equal margins. :)